docs/testplan/token-payment-processor-testplan.md

```diff
@@ -0,0 +1,49 @@
+  | Update Geth Node | Geth nodes can get outdated and this may lead to further complications, so for this case, if there are new patches or updates of geth then we should be alerted to update | Must have early on
+  | Resynced Geth Node | Geth nodes should be able to sync with the network, if a geth node resyncs it can take several days and so the token payment processor should be alerted and pick up transactions during this resync | Must have early on
+UI | Confirmed/Paid Transaction | Users should be able to see if a transaction is confirmed and paid for on UI | Must have early on
+  | Pending Transaction | Users should be able to see if a transaction is pending on UI | Must have early on
+  | Failed Transaction | Users should be able to see if a transaction failed on UI | Must have early on
+  | Transaction Override UI | Users should be able to see if a transaction is succesfully overrided on UI (so user should see status go from pending-->confirmed, during the process of transaction override) | Must have early on
+  | Transaction Ordered By Nonce/ View Nonce | When a user performs multiple transactions using the same address, those with a lower nonce (older transactions) are included in the blockchain first for security purposes regardless of any issue or User should be able to view Nonce | Can be implemented later
+  | Balance Increase | Balance increases should be based on the first confirmation and not based on the time we process it | Must have early on (viewable from transaction details link from wallet))
+  | Sender Fee | Sender fee should be obtainable from UI for said transaction | Must have early on (viewable from transaction details link from wallet))
+  | Reciever Fee | Reciever fee should be obtainable from UI for said transaction | Must have early on (viewable from transaction details link from wallet))
+  | Sender/Reciever Address | Sender and receiver address should be obtainable from UI for said transaction | Must have early on (viewable from transaction details link from wallet))
+  | Transaction Fee | Price from gas fee, gas used from transaction etc should be obtainable from UI | Must have early on (viewable from transaction details link from wallet)
+  | Conversion Rate | When user adds storj token as payment method, token should convert to USD in total balance with 10 percent bonus added on top | Token should convert to USD otherwise it can cause some problems f.e person wants to charge back when token buying power increases
+  | Transaction Detail | If user makes a transaction user should be able to see ethereum scaling solution used for said transaction (view details) | Must have early on
+Testing | Recreate Transactions | Using sender, receiver, nonce, and transaction fees we should be able to recreate transactions to see if there are any mempool bugs and this should also be doable using storj-up | Can be implemented later
```

Comment:
I would not say that we can implement this later on. We need a way to run tests early on.